.github/workflows/test.yml

```diff
@@ -118,10 +118,18 @@ jobs:
       - name: "Test building C bindings"
         shell: bash
         run: |
+          set -e
           mkdir examples/ffi/build
           cd examples/ffi/build
-          cmake -S .. -DDatadog_ROOT=$OUTPUT_FOLDER/profiling
-          cmake --build .
+          # Add BUILD_SYMBOLIZER variable only for Linux platforms
+          if [[ "${{ matrix.platform }}" == "ubuntu-latest" ]]; then
+            cmake -S .. -DDatadog_ROOT=$OUTPUT_FOLDER/profiling -DBUILD_SYMBOLIZER=true
```

Comment:
I think this could be true. 
Having C generation and C tests within a build script sounds good.

If that is OK, I would prefer pushing this to the next PR.